components/feature/addons/src/main/java/mozilla/components/feature/addons/Addon.kt

```diff
@@ -121,6 +122,11 @@ data class Addon(
      */
     fun isSupported() = installedState?.supported == true
 
+    /**
+     * Returns whether or not this [Addon] is currently disabled because it is not supported.
```

Comment:
nit: comment mentioning this is based on [Engine]

Reply:
Updated!